### PR TITLE
E2E Test: Suggestion Input

### PR DESCRIPTION
**Pull request by aaschlote**

### CODE REVIEW

File: tests/e2e/pom/Vorabcheck.ts [hunk 1]
```diff
@@ -26,30 +26,62 @@ export class Vorabcheck {
     await this.page.locator(`label[for="${field}-${option}"]`).click();
   }
 
-  async clickNext() {
+  async clickNext(javaScriptEnabled: boolean) {
```
Comment (chohner) [inline tests/e2e/pom/Vorabcheck.ts:29]: this is a rather surprising argument, maybe it should rather be in an named argument?

File: tests/e2e/pom/Vorabcheck.ts [hunk 2]
```diff
@@ -26,30 +26,62 @@ export class Vorabcheck {
...
     await this.page.locator(`button[name="${this.nextButtonName}"]`).click();
-    await this.page.waitForNavigation(); // deprecated but URL for waitForURL is unknown
+    if (javaScriptEnabled) {
```
Comment (chohner) [inline tests/e2e/pom/Vorabcheck.ts:31]: I'm confused how this actually does anything for JS on the Page? 😅

Comment (aaschlote) [reply tests/e2e/pom/Vorabcheck.ts:31]: The ``waitForNavigation`` has a parameter called ``waitUntil``, where all the methods needs JS on the page, example ``load``. More info in this [link](https://playwright.dev/docs/api/class-page#page-wait-for-url)

File: tests/e2e/pom/Vorabcheck.ts (hunk 1 above)
Comment (aaschlote) [reply tests/e2e/pom/Vorabcheck.ts:29]: It sounds a good idea, I'll do it :)

**Pull request merged**